adobecom/dc
# MWPW-137866: Prevent Widget disappearing on file select - CLS Improvement

#### Changes Made:

- Addressed the CLS (Cumulative Layout Shift) issue related to the widget disappearing briefly when a user tries to upload a file immediately after clicking the main CTA.
- The white box widget now remains visible during file processing, preventing CLS spikes.

#### How to Test:

1. Throttle your Chrome to slow 3G or fast 3G.
2. Visit one of the "problem" pages, e.g., https://www.adobe.com/acrobat/online/compress-pdf.html.
3. Wait for the main CTA to become enabled (no longer grey).
4. Click on the CTA and try to upload a file.
5. Verify that the widget no longer disappears and that there is no significant CLS spike.

#### Additional Information:

Resolves: [MWPW-139699](https://jira.corp.adobe.com/browse/MWPW-139699)

Test URLs:

Before: https://www.adobe.com/acrobat/online/compress-pdf.html?martech=off
After: https://mwpw-139699--dc--adobecom.hlx.live/acrobat/online/compress-pdf?martech=off

#### Screenshots:

<img width="423" alt="Screenshot 2023-12-07 at 11 48 26 AM" src="https://github.com/adobecom/dc/assets/194010/0909e3da-fae9-468a-b284-9b18662a9a58">
